README.md

```diff
@@ -200,9 +200,14 @@ name>-<application name>` to see more information.
   * Enable [monitoring for user-defined projects](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/monitoring/enabling-monitoring-for-user-defined-projects) in OpenShift clusters
     * `oc -n openshift-monitoring edit configmap cluster-monitoring-config`
     * Set variable `enableUserWorkload` to `true`
+  * Install the `Red Hat Build of Opentelemeter` from the operatorhub
 * Core/Hub cluster
+  * Login to the core cluster using an account that has cluster-admin privileges
   * Edit contents of [thanos-secret](acm/odh-core/acm-observability/secrets/thanos.yaml) file.
-  * Install the ACM observability stack by running `make install`
+  * Install the ACM observability stack by running `make setup-observability`
+  * Create the MCO pull secret from the cluster secret
```

Comment:
Just to prevent confusion can we use the object name instead of the `MCO`?

Reply:
:+1: